src/compton/config.h.in

```diff
@@ -14,6 +14,7 @@
 #ifdef COMPTON_FOUND
 // headers provided in compton CSK_generator include directory
 #include "etemp_interp.hh"
+#include "llnl_interp.hh"
```

Comment:
We might want to consider installing these headers into `$prefix/include/csk` so that these include directives are a bit more *self-documenting*.  That is, `include "csk/llnl_interp.hh"` is a bit more descriptive for someone unfamiliar with this package.

Reply:
Sure, I can do that! :)